pkg/resource/job.go

```diff
@@ -106,14 +107,14 @@ func (b JobBuilder) MakeContainers() []corev1.Container {
 			ImagePullPolicy: *b.Spec().Image.PullPolicyName,
 			Resources:       b.Spec().Resources,
 			Command:         []string{"/bin/bash"},
-			Args:            []string{"-c", fmt.Sprintf("%s; sleep 120", GetTagVersionCommand)},
+			Args:            []string{"-c", fmt.Sprintf("%s; sleep 150", GetTagVersionCommand)},
```

Comment:
Can we pull the logs from the pod even if the job is done? The timing of getting the logs vs running the job is bugging me.

Reply:
I added 150 sec for the job to complete and 120 sec to pull the version. If the version checker retries a new job is created, so we do not have the case where we try to pull the logs from  a pod when the job is completed.